snooty.toml

```diff
@@ -9,5 +9,6 @@ toc_landing_pages = [
 
 [constants]
 version = 4.3
+minor-version = "{+version+}.2"
```

Comment:
Nit: Semver is <major>.<minor>.<patch>  So the minor version is 3 and the patch is 2

Perhaps call the constant full-version ?